puppet/modules/site_couchdb/manifests/init.pp

```diff
@@ -41,8 +41,8 @@
   $couchdb_pwhash_alg       = $couchdb_config['pwhash_alg']
 
   if $couchdb_mode == 'multimaster' { include site_couchdb::bigcouch }
-  if $couchdb_mode == 'master'      { include site_couchdb::master }
-  if $couchdb_mode == 'mirror'      { include site_couchdb::mirror }
+  if $couchdb_mode == 'plain'       { include site_couchdb::plain }
+  # if $couchdb_mode == 'mirror'      { include site_couchdb::mirror }
```

Comment:
the way it was supposed to work before was that 'couch.mode' was automatically set, and the sysadmin would configure 'couch.master'. if there is a single node with 'couch.master' set to true, then it will still use plain couch. see facbf937a4694c775c4f1c92176cd3f513c21d9a